regression/cbmc/pointer-offset-01/test-no-cp.desc

```diff
@@ -0,0 +1,11 @@
+FUTURE
+main.c
+--no-simplify --no-propagation
+^EXIT=10$
+^SIGNAL=0$
+\[main.assertion.1\] line \d+ assertion __CPROVER_POINTER_OFFSET\(p\) >= 0: FAILURE
+--
+^warning: ignoring
+--
+Check that a negative offset can be chosen for uninitialized pointers. Currently
```

Comment:
Again - I sort of feel like the correct behavior for something like this is, asserts always fail? Like the offset isn't definitely positive, but also isn't definitely negative - it should be havoc'd to be any value

Reply:
Added